src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -251,6 +252,8 @@ private void InitializeCore()
             InsertClassesIntoTree(LibraryRootCategories);
 
             ChangeRootCategoryExpandState(BuiltinNodeCategories.GEOMETRY, true);
+
+            selectionNavigator = new SelectionNavigator(SearchRootCategories);
```

Comment:
Hi @aosyatnik, please move this all the way up to constructor so that we won't accidentally access a `null` selection navigator.